harvard-huit/Harvard-Geospatial-Library
# Updated harvard-patterns-gem with all HGL styles

**Updated harvard-patterns-gem with all HGL styles**
* * *

**JIRA Ticket**: [Harvard Library Theme Gem](https://jira.huit.harvard.edu/browse/HGL-298)

# What does this Pull Request do?
Stylesheets local to HGL have been removes and placed in harvard-patterns-gem.
Contact email in footer has been updated.

# How should this be tested?
* Rebuild the docker container
* Check to see that container is `Using harvard-patterns-gem 1.0 from https://gitlab.com/harvard-library-web-team/harvard-patterns-gem.git (at 1.0@3b93e8b)`
* All styles should look and work as they do on dev and QA
* Check that gem folder is removed from HGL repo (app/assets/stylesheets/gem should no longer exist)
* Check that "Contact Us" link in the footer points to `hgladmin@fas.harvard.edu`

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no

# Interested parties
@phil-plencner-hl 
@enriquediaz
